specification/sql/resource-manager/Microsoft.Sql/preview/2022-11-01-preview/LongTermRetentionPolicies.json

```diff
@@ -186,10 +186,57 @@
     }
   },
   "definitions": {
-    "BaseLongTermRetentionPolicyProperties": {
```

Comment:
This breaking change for the property is expected? It has triggered the breaking change pipeline failed.